package.json

```diff
@@ -69,18 +73,25 @@
     "babel-loader": "^8.2.2",
     "babel-polyfill": "^6.26.0",
     "codemirror": "^5.58.3",
+    "core-js": "^3.8.1",
```

Comment:
What're you using core-js for?

Reply:
Trying to get `async`/`await` support? I was opting not to use `babel-polyfill` since it's deprecated. In that case, I should probably replace `babel-polyfill` elsewhere in the repo?